enterprise/frontend/src/metabase-enterprise/hosting/index.js

```diff
@@ -10,14 +9,6 @@ import { SettingsCloudStoreLink } from "./components/SettingsCloudStoreLink";
 if (hasPremiumFeature("hosting")) {
   PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections => _.omit(sections, ["updates"]));
 
-  PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections =>
-    updateIn(sections, ["general", "settings"], settings =>
-      _.reject(settings, setting =>
-        ["site-url", "redirect-all-requests-to-https"].includes(setting.key),
-      ),
-    ),
-  );
-
```

Comment:
here's a crazy bit of code. to remove settings we had to manipulate this array of settings and remove some if the "hosting" token feature was present. Now we can put this logic in the components that actually render these settings so you don't have to run all over the codebase to figure out what is causing those settings to disappear.